pkg/reconciler/revision/revision_test.go

```diff
@@ -488,118 +492,64 @@ func TestNoQueueSidecarImageUpdateFail(t *testing.T) {
 }
 
 func TestGlobalResyncOnConfigMapUpdateRevision(t *testing.T) {
```

Comment:
Looks like this test is now testing updating logging URL. How does this test diff from `TestUpdateRevWithWithUpdatedLoggingURL`? Can one of these be eliminated?